invokeai/app/services/model_manager_service.py

```diff
@@ -396,7 +397,7 @@ def model_exists(
             model_type,
         )
 
-    def model_info(self, model_name: str, base_model: BaseModelType, model_type: ModelType) -> dict:
+    def model_info(self, model_name: str, base_model: BaseModelType, model_type: ModelType) -> Union[dict, None]:
```

Comment:
`Optional`?

Reply:
It's the exact same thing internally, I don't mind either way

Reply:
`Union[dict, None]` keeps the same style for when we need to do eg `Union[datetime, str, None]`, but that's not a big reason.

This is my habit coming from TS where a union w/ none is different than optional. Also, we have undefined for "unvalued" and null for "user-defined unvalued", so that has impacted my style here

Reply:
That's fair. To me `Optional` is synonymous and made for this particular case, but this works too.

Reply:
I'll leave it as it is now, given these particular changes were actually from a different PR in which I fixed a few typing issues with the model manager. Thanks for calling it out regardless.